pkg/settlement/swap/transaction/transaction.go

```diff
@@ -154,6 +154,13 @@ func (t *transactionService) WaitForReceipt(ctx context.Context, txHash common.H
 	}
 }
 
+func IncreaseGasPrice(gasPrice *big.Int) *big.Int {
+	// increase 40%
```

Comment:
how come it becomes `40%` all of a sudden?

Reply:
I see, looks like the official swarm dev team sucks terribly on math lol

Reply:
This is not an official problem, because everyone uses the same value, and transactions are difficult to link in a crowded situation of ethereum.